src/AppInstallerCLICore/AppInstallerCLICore.vcxproj

```diff
@@ -368,6 +368,7 @@
     <ClInclude Include="Commands\HashCommand.h" />
     <ClInclude Include="Commands\ListCommand.h" />
     <ClInclude Include="Commands\PinCommand.h" />
+    <ClInclude Include="Commands\RepairCommand.h" />

```

Comment:
nit: Missing the AppInstallerCLICore.vcxproj.filters file change for putting them under correct filters